assets/styles/styles.css

```diff
@@ -0,0 +1,115 @@
+/* ====== GLOBAL STYLE START ==== */
+
+*,
+*::after,
+*::before {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  min-width: 375px;
+  padding: 0;
+  margin: 0;
+  font-family:
+    "Roboto",
+    "Crete Round",
+    "Poppins",
+    "Helvetica",
+    "Arial",
+    "Verdana",
+    "Tahoma",
+    sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: var(--white);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+small {
+  padding: 0;
+  margin: 0;
+}
+
+.btn-scale:hover {
+  transform: scale(1.3);
+}
+
+/* ====== GLOBAL STYLE END ==== */
+
+/* ====== GLOBAL VARIABLES START ==== */
+
+:root {
+  --primary-color: #ff6b00;
+  --black: #1c1a19;
+  --white: #fff;
+  --light-yellow: #fff5e1;
+  --light-grey: #dbd8d7;
+}
+
+/* ====== GLOBAL VARIABLES END ==== */
+
+/* ====== HEADER START ==== */
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background: #3c3a39;
+  padding: 1.5em 1em;
+}
+
+header .my-logo {
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+/* ====== HEADER END ==== */
+
+/* ====== HERO SECTION START ==== */
+
+.hero {
```

Comment:
- [ ] Kindly use the flexbox property here to make your child's content aligned correctly and it will also help you in your next milestone especially when you will do work on the desktop design. Please add the styles here to make it looks good in terms of design.